tensorflow/python/autograph/operators/py_builtins_test.py

```diff
@@ -137,6 +139,21 @@ def test_range_tensor_empty_range(self):
       r = py_builtins.range_(5, constant_op.constant(2))
       self.assertAllEqual(self.evaluate(r), [])
 
+  def test_enumerate(self):
+    self.assertListEqual(list(py_builtins.enumerate_([3,2,1])), [(0, 3), (1, 2), (2, 1)])
+    self.assertListEqual(list(py_builtins.enumerate_([3,2,1], 5)), [(5, 3), (6, 2), (7, 1)])
+    self.assertListEqual(list(py_builtins.enumerate_([-8], -3)), [(-3, -8)])
+
+  def test_enumerate_dataset(self):
+    dataset = dataset_ops.DatasetV2.from_tensor_slices(["a", "c"])
+    start = constant_op.constant(20, dtype=dtypes.int64)
+    dataset = dataset.enumerate(start)
```

Comment:
We want `py_builtins.enumerate(dataset, start)` here, right?

Reply:
Yes, I've changed it on my recent commit.